configure.ac

```diff
@@ -320,6 +320,18 @@ AC_ARG_WITH([systemdsystemunitdir],
 	[systemdsystemunitdir=$($PKG_CONFIG --variable=systemdsystemunitdir systemd)])])
 AC_SUBST([SYSTEMD_SYSTEM_UNIT_DIR], [$systemdsystemunitdir])
 
+AC_ARG_WITH([systemdbluealsaargs],
+	AS_HELP_STRING([--with-systemdbluealsaargs], [profiles to be enabled in bluealsa.service, defaults to '-p a2dp-source -p a2dp-sink' if not specified]),
```

Comment:
Update help string to reflect purpose (not only profiles). Also, you do not have to check for `AM_COND_IF([ENABLE_SYSTEMD]`. This check is required in `--with-systemdsystemunitdir` because later we are reading data from systemd pkg-config file. That's not the case for this option.

Reply:
Thanks for the review, have pushed a V3. @Arkq